Dispatcher/Constants/Constants.swift

```diff
@@ -46,6 +46,11 @@ struct Constants {
         static let newsUrl = "https://api.newscatcherapi.com/v2/search"
     }
     
+    struct pageSizeToFetch {
+        static let savedArticles = 10
```

Comment:
We usually naming static vars with uppercase to distinguish between them and run time vars.
You rename all your static vars here with uppercase. Ex: NEWS_API_KEY  = ""

Reply:
Fixed for all variables in the Constants file
@abenartz